src/game/boe.main.cpp

```diff
@@ -135,21 +149,64 @@ int main(int argc, char* argv[]) {
 	}
 }
 
-static void init_sbar(std::shared_ptr<cScrollbar>& sbar, rectangle rect, int max, int pgSz, int start = 0) {
+static void init_sbar(std::shared_ptr<cScrollbar>& sbar, std::string const name, rectangle rect, rectangle events_rect, int max, int pgSz, int start = 0) {
```

Comment:
Oh, "events_rect" is nice, maybe _that_ can be used as its name within the cScrollbar class. Or something more similar to it.